addons/social_share/wizards/snshare_url.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
```

Comment:
Yes, utf8 is the default since a while and license is already in the module/at the root of the repo so those can be omitted.

We don't really validate xml files either afaik, so will remove utf-8 from them for consistency